openpype/settings/entities/root_entities.py

```diff
@@ -755,6 +755,35 @@ def defaults_dir(self):
         """
         return DEFAULTS_DIR
 
+    def settings_value(self):
+        output = super(ProjectSettings, self).settings_value()
+
+        anatomy = output.get(PROJECT_ANATOMY_KEY) or {}
+
+        # Evaluate project archiving flag
+        #
+        archive_confirm = anatomy.get("attributes", {}).get("archive_confirm")
```

Comment:
Do not set unset keys by default. That way the key won't be marked as modified (which is because have missing key) and won't show you that project has missing set values...